frontend/src/components/common/Pagination.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useEffect } from "react";
+import { extendTheme, chakra, ChakraProvider, Box } from "@chakra-ui/react";
+import {
+  Pagination as P,
+  usePagination,
+  PaginationNext,
+  PaginationPage,
+  PaginationPrevious,
+  PaginationContainer,
+  PaginationPageGroup,
+  PaginationSeparator,
+} from "@ajna/pagination";
+import paginationStyle from "../../themes/components/pagination";
+
+const customTheme = {
+  ...chakra,
+  fonts: {
+    body: '"DM Sans"',
+    heading: '"DM Sans"',
+    mono: '"DM Sans"',
+  },
+  colors: {
+    brand: {
+      900: "#154472",
+    },
+  },
+  components: {
+    paginationStyle,
+  },
+};
+
+const outerLimit = 1;
+const innerLimit = 1;
+
+interface PaginationProps {
+  pagesCount: number;
+  currentPage: number;
+  itemsToShow: number;
+  onPageChange: (page: number) => void;
+}
+
+const Pagination = ({
+  pagesCount,
+  currentPage,
+  onPageChange,
+  itemsToShow,
+}: PaginationProps): React.ReactElement => {
+  const { pages } = usePagination({
+    pagesCount,
+    initialState: { currentPage },
+  });
+
+  return (
+    <ChakraProvider theme={customTheme}>
+      <Box display="flex" justifyContent="center" style={{ height: "75px" }}>
+        <P
+          pagesCount={pagesCount}
+          currentPage={currentPage}
+          onPageChange={onPageChange}
+        >
+          <PaginationContainer>
+            {currentPage !== 1 && (
+              <PaginationPrevious
+                bg="transparent"
+                color="#636363"
+                fontWeight="normal"
+                border="none"
+                textDecoration="underline"
+                _hover={{ textDecoration: "underline", fontWeight: "600" }}
+                style={{
+                  margin: "0 10px",
+                }}
+                outline="none"
+              >
+                &lt; Previous
+              </PaginationPrevious>
+            )}
+            <PaginationPageGroup isInline align="center">
+              {pages.map((page: number, index: number) => {
+                if (
+                  index === 0 ||
+                  index === pages.length - 1 ||
+                  (index >= currentPage - innerLimit - 1 &&
+                    index <= currentPage + innerLimit - 1)
+                ) {
+                  return (
+                    <PaginationPage
+                      bg="#E8EDF1"
+                      key={`pagination_page_${page}`}
+                      page={page}
+                      onClick={() => console.warn("Im clicking the page")}
+                      color="#636363"
```

Comment:
as explained above, please move all this into `themes/components/button.ts` and create a new variant and size

then you can do smth like
```
<PaginationPage
   size="paginationSize"
   variant="pagination"
```